<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>477</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;云南大学学生排班工具&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pb_view_t">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
            </property>
            <property name="text">
             <string>查看课表</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pb_view_free_t">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
            </property>
            <property name="text">
             <string>查看空闲表</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="pb_import_t">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
            </property>
            <property name="text">
             <string>导入课表</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="topMargin">
           <number>50</number>
          </property>
          <item>
           <widget class="QPushButton" name="pb_gen_duty_t">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
            </property>
            <property name="text">
             <string>生成排班表</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="bottomMargin">
           <number>50</number>
          </property>
          <item>
           <widget class="QPushButton" name="pb_gen_free_t">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
            </property>
            <property name="text">
             <string>生成空闲表</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
